src/Sulu/Bundle/TranslateBundle/Translate/Export.php

```diff
@@ -233,64 +213,84 @@ public function getPath()
     }
 
     /**
-     * @param int $packageId
+     * Executes the export by loading the catalogue and writing messages
+     * from the defined domains to the defined file.
      */
-    public function setPackageId($packageId)
+    public function execute()
     {
-        $this->packageId = $packageId;
+        $messages = [];
```

Comment:
Sry, have overseen this one before... This should actually be also `null`, and in the `if`s below there should be no `array_merge`.